packages/overlays/src/LocalOverlayController.js

```diff
@@ -145,6 +149,9 @@ export class LocalOverlayController {
       this.contentNode.style.zIndex = 1;
       this.invokerNode.setAttribute('aria-expanded', true);
 
+      if (this.inheritsReferenceObjectWidth) {
+        this.contentNode.style.width = `${this.invokerNode.getBoundingClientRect().width}px`;
```

Comment:
We should also think about min width and max width here

Reply:
I will investigate this